batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/dao/UserDAOImpl.java

```diff
@@ -0,0 +1,143 @@
+package com.lftechnology.batch7crud.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.lftechnology.batch7crud.db.DbConnector;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.util.TypeCaster;
+
+public class UserDAOImpl implements UserDAO {
+
+    Connection connection = null;
+
+    public UserDAOImpl() {
+        try {
+            this.connection = DbConnector.getMySqlConnection();
+        } catch (SQLException e) {
+            // TODO Auto-generated catch block
+            e.printStackTrace();
+        }
+
+    }
+
+    public void add(User user) throws DataException {
+        try {
+            String str = "insert into user (firstname,surname,username,password) values (?,?,?,?)";
+            PreparedStatement statement = connection.prepareStatement(str);
+
+            statement.setString(1, user.getFirstName());
+            statement.setString(2, user.getSurName());
+            statement.setString(3, user.getUserName());
+            statement.setString(4, user.getPassword());
+
+            statement.execute();
+            System.out.println("user added");
+        } catch (SQLException e) {
+            System.err.println(e);
+            throw new DataException();
+        }
+    }
+
+    public void delete(int userID) throws DataException {
+        try {
+            String query = "delete from user where id= ?";
+            PreparedStatement statement = connection.prepareStatement(query);
+
+            statement.setLong(1, userID);
+            statement.executeUpdate();
+
+            System.out.println("user deleted");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)